pootle/apps/pootle_store/diff.py

```diff
@@ -51,32 +51,37 @@ def target(self):
 
 class StoreDiff(object):
 
-    def __init__(self, db_store, file_store, file_revision):
-        self.db_store = db_store
-        self.file_store = file_store
-        self.db_revision = db_store.get_max_unit_revision()
-        self.file_revision = file_revision
+    def __init__(self, target_store, source_store, source_revision):
```

Comment:
@ta2-1 in this case i think yes - because the target is the thing to be updated - and the update code does something similar

not sure tho - thought about this quite a bit and didnt come to a firm conclusion.

also worth mentioning - this is how it is now - just that it uses more confusing vars